certbot-apache/certbot_apache/augeasparser.py

```diff
@@ -152,12 +152,21 @@ def find_directives(self, name, exclude=True): # pylint: disable=unused-argument
         return nodes
 
     def find_comments(self, comment, exact=False): # pylint: disable=unused-argument
-        """Recursive search of DirectiveNodes from the sequence of children"""
-        new_metadata = {"augeasparser": self.parser}
-        return [AugeasCommentNode(comment=assertions.PASS,
-                                  ancestor=self,
-                                  filepath=assertions.PASS,
-                                  metadata=new_metadata)]
+        """
+        Recursive search of DirectiveNodes from the sequence of children.
+
+        Note that the argument exact is not implemented here, as it's not
+        currently in use in ApacheConfigurator.
```

Comment:
now that we're actually implemented, the comment should describe the arguments and return values